hw2/02-reverse.js

```diff
@@ -1,3 +1,49 @@
-/** Exercise 02 - Reverse **/
+/** 
+ * Exercise 02 - Reverse 
+ * Brooke Czerwinski
+ * Full-Stack Web Development
+ * HW 2
+ *
+ * References
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp
+ * https://www.codeproject.com/Questions/1232562/I-want-to-display-form-data-on-same-page-using-jav
+ * https://www.geeksforgeeks.org/reverse-a-string-in-javascript/ (I think method 2 is really nice!)
+**/
 
-// Add your code here
+// Function to reverse string
+function reverseNumber(input) {
```

Comment:
need to reverse number as number not as string

Reply:
There is no requirement that I can see in the homework that the reversed output be a number. As a string it is easier to work with and can provide the requested output. Additionally, if this number were to be used in some math function, I could easily translate it back from a string into a number using `parseInt()`, though that step seems unnecessary here.

Reply:
In the HW2, it was mentioned :-
Write a function that reverses the digits in a number.

Reply:
Hm, I didn't read it as meaning a datatype, but it's possible that's true and I just misunderstood. Maybe @JeremyIV can confirm one way or the other if the user input needs to remain a number or if it's ok to take in a number but then transform it into a string to reverse it?

Reply:
I would consider this to be an implementation detail up to your discretion as a programmer. This problem is about the user interface more than doing some kind of clever numerical manipulations in Javascript. I think treating the number as a string is a very reasonable way to reverse it.

Reply:
Thank you!